### DISCUSSION TITLE
Are there any closed captioning implementations

### DISCUSSION THREAD

**Discussion by ccreedmanor** [openai/whisper#1598] [Q&A]
Looking for a project that functions similarly to https://webcaptioner.com/, trying to provide closed captioning to our livestreams.

**Comment by glangford** [comment]
See the discussions here for some ideas.

* https://github.com/openai/whisper/discussions/2
* https://github.com/openai/whisper/discussions/458
* https://github.com/openai/whisper/discussions/1422
